android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/PrimaryTextField.kt

```diff
@@ -0,0 +1,72 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color.Companion.Transparent
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.text.input.*
+import androidx.compose.ui.tooling.preview.Preview
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.shapes
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.typography
+import vn.luongvo.kmm.survey.android.ui.theme.White18
+import vn.luongvo.kmm.survey.android.ui.theme.White30
+
+@Composable
+fun PrimaryTextField(
+    value: String,
+    onValueChange: (String) -> Unit,
```

Comment:
@luongvo I think this property should be `optional`. 🤔

Reply:
@doannimble it's a required field from the original TextField so I would like to follow that constraint. Actually we need to catch all input events to get its value 😄 

![image](https://user-images.githubusercontent.com/16315358/208351520-0647dcd3-2d4c-48a7-8e16-ad5cb8daa756.png)

Reply:
@luongvo Thanks for the clarification. In some cases, we only need to get the final text without checking the text change.
We're defining the common, so in my humble opinion, forcing it is not a good idea. But anyway, we should keep it as the original TextField is required.

Reply:
@doannimble Note that, Jetpack Compose is far from the XML method. I'm wondering how can we get the input value without having `onValueChange` https://developer.android.com/jetpack/compose/text#enter-modify-text 😄